content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -40,21 +96,37 @@ All resource types are either scoped by the cluster (`/apis/GROUP/VERSION/*`) or
 
-Since a namespace is a cluster-scoped resource type, you can retrieve the list of all namespaces with `GET /api/v1/namespaces` and details about a particular namespace with `GET /api/v1/namespaces/NAME`.
-
-Almost all object resource types support the standard HTTP verbs - GET, POST, PUT, PATCH, and DELETE. Kubernetes uses the term **list** to describe returning a collection of resources to distinguish from retrieving a single resource which is usually called a **get**.
-
-Some resource types will have one or more sub-resources, represented as sub paths below the resource:
+Since a namespace is a cluster-scoped resource type, you can retrieve the list
+(“collection”) of all namespaces with `GET /api/v1/namespaces` and details about
+a particular namespace with `GET /api/v1/namespaces/NAME`.
 
 * Cluster-scoped subresource: `GET /apis/GROUP/VERSION/RESOURCETYPE/NAME/SUBRESOURCE`
 * Namespace-scoped subresource: `GET /apis/GROUP/VERSION/namespaces/NAMESPACE/RESOURCETYPE/NAME/SUBRESOURCE`
 
-The verbs supported for each subresource will differ depending on the object - see the API documentation for more information. It is not possible to access sub-resources across multiple resources - generally a new virtual resource type would be used if that becomes necessary.
+The verbs supported for each subresource will differ depending on the object -
+see the [API reference](/docs/reference/kubernetes-api/) for more information. It
+is not possible to access sub-resources across multiple resources - generally a new
+virtual resource type would be used if that becomes necessary.
 
 
 ## Efficient detection of changes
 
-To enable clients to build a model of the current state of a cluster, all Kubernetes object resource types are required to support consistent lists and an incremental change notification feed called a **watch**.  Every Kubernetes object has a `resourceVersion` field representing the version of that resource as stored in the underlying database. When retrieving a collection of resources (either namespace or cluster scoped), the response from the server will contain a `resourceVersion` value that can be used to initiate a watch against the server. The server will return all changes (creates, deletes, and updates) that occur after the supplied `resourceVersion`. This allows a client to fetch the current state and then watch for changes without missing any updates. If the client watch is disconnected they can restart a new watch from the last returned `resourceVersion`, or perform a new collection request and begin again. See [Resource Version Semantics](#resource-versions) for more detail.
+To enable clients to build a model of the current state of a cluster, all Kubernetes
+object resource types are required to support consistent **list** requests and an
+incremental change notification feed called a **watch**.  Every Kubernetes object
```

Comment:
Watch is not mandatory for aggregated apiservers, and especially for virtual resources. (and in fact is super hard to implement).

Reply:
Reworded.